README.md

```diff
@@ -25,132 +21,112 @@ Package.onUse(function (api) {
 ```
 
 ## Compatibility
-<table>
-<thead>
-<tr><th>Meteor Version</th><th>Recommended fourseven:scss version</th></tr>
-</thead>
-<tbody>
-<tr><td>1.0 - 1.1</td><td>3.2.0</td></tr>
-<tr><td>1.2 - 1.3.1</td><td>3.4.2</td></tr>
-<tr><td>1.3.2+</td><td>3.8.0_1</td></tr>
-<tr><td>1.4.0</td><td>3.8.1</td></tr>
-<tr><td>1.4.1+</td><td>4.5.4</td></tr>
-<tr><td>1.6+</td><td>4.12.0</td></tr>
-</tbody>
-</table>
```

Comment:
@illusionfield I'm suggesting to keep this table for historical reference and to provide support for older projects